sdk/keyvault/perf-tests/keyvault-keys/test/cryptography/cryptography.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { PerfStressTest } from "@azure/test-utils-perfstress";
+import { CryptographyClient, KeyClient } from "@azure/keyvault-keys";
+import { v4 as uuid } from "uuid";
+import { credential, keyVaultUri } from "../utils";
+
+export abstract class CryptographyTest<TOptions = {}> extends PerfStressTest<TOptions> {
+  keyClient: KeyClient;
+  static cryptoClient?: CryptographyClient;
+  static keyName = `k-${uuid()}`;
+
+  constructor() {
+    super();
+    this.keyClient = new KeyClient(keyVaultUri, credential);
+  }
+
+  async globalSetup() {
+    // Create a single shared key for all tests
+    const key = await this.keyClient.createRsaKey(CryptographyTest.keyName, {
+      keySize: 2048
```

Comment:
Consider making the key size a parameter to the test. Also please revisit other tests and see if similar constants could be made parameters as well.